vowpalwabbit/vw.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14

```

Comment:
This retargets us to VS2017 instead of VS2015. Is anyone against this? The new style C# project structure requires this.

Reply:
If it's working I'm very happy to upgrade to latest version